### PR TITLE
Searchbar changes

### LINKED ISSUE #26: remove default inputs from search bar (effort 1)
image

we dont need default values there

### PR DESCRIPTION
**Pull request by hoolek77**
removed default values except adult and children count
fixed request as well (whenever user doesnt provide date the request is made without it in query params)
added responsivness

image

### CODE REVIEW

File: src/hooks/useSearch.js [hunk 1]
```diff
@@ -5,14 +5,23 @@ export const useSearch = () => {
   const [hotels, setHotels] = useState([])
 
   const search = async ({ city, startDate, endDate, adults, children }) => {
```
Comment (GoraJakub) [inline src/hooks/useSearch.js:7]: Add try and catch

**Review by GoraJakub — CHANGES_REQUESTED** [review]

File: src/hooks/useSearch.js [hunk 2]
```diff
@@ -5,14 +5,23 @@ export const useSearch = () => {
...
-        endDate: new Date(endDate).toISOString(),
+        startDate,
+        endDate,
         adults,
         children,
-      },
+      }
+    }
+
+    const response = await axios.get(global.API_BASE_URL + 'api/hotels', {
```
Comment (pdybowski) [inline src/hooks/useSearch.js]: Here you should add try and catch Block, not above

File: src/hooks/useSearch.js (hunk 1 above)
Comment (pdybowski) [reply src/hooks/useSearch.js:7]: A little bit Lower

File: src/hooks/useSearch.js (hunk 2 above)
Comment (GoraJakub) [reply src/hooks/useSearch.js]: That's right

File: src/hooks/useSearch.js [hunk 3]
```diff
@@ -20,11 +20,15 @@ export const useSearch = () => {
...
-    })
+    try {
+      const response = await axios.get(global.API_BASE_URL + 'api/hotels', {
+        params: data,
+      })
 
-    setHotels(response.data)
+      setHotels(response.data)
+    } catch (ex) {
+      console.log(ex)
```
Comment (pdybowski) [inline src/hooks/useSearch.js]: I will add alert here. Is it exception or exception.message?

**Pull request merged**